src/main/kotlin/org/rust/lang/core/psi/ext/RsForeignModItem.kt

```diff
@@ -21,7 +21,7 @@ abstract class RsForeignModItemImplMixin : RsStubbedElementImpl<RsPlaceholderStu
     constructor(stub: RsPlaceholderStub, elementType: IStubElementType<*, *>) : super(stub, elementType)
 
     override val outerAttrList: List<RsOuterAttr>
-        get() = childrenOfType()
+        get() = PsiTreeUtil.getStubChildrenOfTypeAsList(this, RsOuterAttr::class.java)
```

Comment:
Maybe extract it as separate function like `childrenOfType`

Reply:
Can we just change `PsiElement.childrenOfType` to call `getStubChildrenOfTypeAsList` instead of `getChildrenOfTypeAsList` ? Or is there any drawback to such a change?

Reply:
No, we can't replace `childrenOfType` by `getStubChildrenOfTypeAsList` because if element is `StubBasedPsiElement`, `getStubChildrenOfTypeAsList` returns only inheritors of `StubBasedPsiElement` while `childrenOfType` returns all children.
For example, if `element` is `RsArrayType`, result of `PsiTreeUtil.getStubChildrenOfTypeAsList(element, RsElement::class.java)` doesn't contain expr child because `RsExpr` is not `StubBasedPsiElement`

Reply:
Ah OK. I have created a wrapper `PsiElement.stubChildrenOfType()`